modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,218 @@
+= Recovery Mode
+:description: Recovery mode starts Redpanda in "metadata only" mode and disables partitions so you can repair a failed cluster.
+
+Recovery mode allows you to start Redpanda and form a stable cluster when it fails due to a system crash, OOM errors, or other issues that make a normal startup infeasible. 
+
+In recovery mode, cluster configuration changes and other manual administrative actions are permitted so you can repair and restore the failed cluster. Topic reads and writes are disabled.
+
+== About recovery mode
```

Comment:
I think this whole section should be renamed. It's listing enabled/disabled operations/functionality. It'd be easier to scan if we had the headings 'Disabled functionality' and 'Enabled functionality'.